styles.css

```diff
@@ -17,13 +17,128 @@ nav {
   padding: 0 24px;
 }
 
-nav .bar1,
-.bar2,
-.bar3 {
+#toggle-icon {
+  display: block;
+}
+
+.works .head-work div {
+  display: flex;
+  flex-direction: column;
+  margin-top: 50px;
+}
+
+#toggle-icon div {
   width: 35px;
-  height: 5px;
+  height: 3px;
   background-color: white;
   margin: 6px 0;
+  transition: 0.4s;
+}
+
+.open .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-6px, 6px);
+  transform: rotate(-45deg) translate(-6px, 6px);
+}
+
+.open .bar2 {
+  opacity: 0;
+}
+
+.open .bar3 {
+  -webkit-transform: rotate(45deg) translate(-6px, -8px);
+  transform: rotate(45deg) translate(-6px, -8px);
+}
+
+nav .nav-menu {
+  display: none;
+  position: absolute;
+  top: 50px;
+  left: 0;
+  height: calc(100vh - 50px);
+  width: 100%;
+}
```

Comment:
- [OPTIONAL] It would be nice if the user cannot scroll down or make the mobile menu sticky so that if the user tries to scroll down while the menu is open, the user cannot accidentally leave the menu creating a better user experience. Now if the user scrolls down after opening the menu, something like this can happen:
 - 
<img width="475" alt="image" src="https://user-images.githubusercontent.com/61919617/223152758-5752627e-bea8-46e4-9db0-4398f2cf7365.png">